hypha/static_src/sass/components/_form.scss

```diff
@@ -38,13 +38,28 @@
             padding-left: 0.5rem;
             position: relative;
             max-width: 100%;
-
             &::before {
                 display: none;
             }
         }
     }
 
+    &--width {
+        flex-grow: 2;
+    }
+
+    &--score-box {
+        display: none;
+        background-color: #1d79a8;
+        color: #fff;
+        padding: 0.5em 60px;
+        font-weight: 700;
+        position: sticky;
+        position: -webkit-sticky;
```

Comment:
Not needed. Webkit supports `sticky` since some years.

Reply:
Lint found the same once I ran it.